### ISSUE TITLE
Error handling

### ISSUE THREAD

**Issue by HenriPodolski** [HenriPodolski/complay#20]
* Display library errors only once, when used with AppFacade
* Method in AppFacade should handle that

E.g. this is logged on every Component creation:
```text
Extension for "dom" is not configured yet.
Please pass an extensions through ApplicationFacade constructor options.dom
or directly through Module, Service or Component via options.app.dom!
```
if dom extension is not configured.

**Comment by HenriPodolski** [comment]
Works at least for standard approach. Won't fix for pre-built es5 package.
